src/ic-cdk-timers/src/lib.rs

```diff
@@ -265,7 +265,12 @@ extern "C" fn timer_executor() {
     if ic_cdk::api::caller() != ic_cdk::api::id() {
         ic_cdk::trap("This function is internal to ic-cdk and should not be called externally.");
     }
-    let (task_id,) = ic_cdk::api::call::arg_data();
+    let config = ic_cdk::api::call::ArgDecoderConfig {
+        decoding_quota: Some(10_000),
+        skipping_quota: Some(100),
```

Comment:
Where do these numbers come from?

Reply:
It's an arbitrary number, but this method only returns a nat, so a small limit on both would be good enough.